### PR TITLE
Update crowdfunding tutorial

### PR DESCRIPTION
**Pull request by danielailie**
#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [ ] document new feature
- [x] update documentation that is not relevant anymore
- [ ] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/developers/tutorials/crowdfunding-p2.md [hunk 1]
```diff
@@ -252,11 +252,11 @@ It doesn't make sense to fund after the deadline has passed, so fund transaction
     fn fund(&self) {
         let payment = self.call_value().egld_value();
 
-        let current_time = self.blockchain().get_block_timstamp();
+        let current_time = self.blockchain().get_block_timestamp();
         require!(current_time < self.deadline().get(), "cannot fund after deadline");
 
         let caller = self.blockchain().get_caller();
-        self.deposit(&caller).update(|deposit| *deposit += payment);
+        self.deposit(&caller).update(|deposit| *deposit += payment.clone_value());
```
Comment (andrei-marinica) [inline docs/developers/tutorials/crowdfunding-p2.md:259]: `self.deposit(&caller).update(|deposit| *deposit += &*payment);`

**Review by andrei-marinica — CHANGES_REQUESTED** [review]

File: docs/developers/tutorials/crowdfunding-p2.md (hunk 1 above)
Comment (danielailie) [reply docs/developers/tutorials/crowdfunding-p2.md]: done

File: docs/developers/tutorials/crowdfunding-p1.md [hunk 2]
```diff
@@ -72,13 +72,13 @@ The source code of each smart contract requires its own folder. You'll need to c
 ```bash
 mkdir -p ~/MultiversX/SmartContracts
 cd ~/MultiversX/SmartContracts
-mxpy contract new crowdfunding --template adder
+mxpy contract new crowdfunding --template empty
```
Comment (andrei-marinica) [inline docs/developers/tutorials/crowdfunding-p1.md:75]: We have a brand new templating tool, released yesterday, going to be improved with the next release today.

Might be a good time to talk about that in the docs.

**Review by andrei-marinica — CHANGES_REQUESTED** [review]

**Review by andrei-marinica — DISMISSED** [review]

File: docs/developers/tutorials/crowdfunding-p1.md (hunk 2 above)
Comment (andreibancioiu) [reply docs/developers/tutorials/crowdfunding-p1.md:75]: When the tool is ready, it will be integrated into `mxpy contract new` :+1:

Comment (andreibancioiu) [inline docs/developers/tutorials/crowdfunding-p1.md:75]: The build is currently failing, but we can fix it with something like this (after invoking `mxpy contract new`):

```
# Temporary workaround, will be fixed soon, when "mxpy" integrates the new templating facility (of "sc-meta").
mv ./crowdfunding/src/empty.rs ./crowdfunding/src/crowdfunding.rs 
```

**Review by andreibancioiu — DISMISSED** [review]

**Review by andreibancioiu — APPROVED** [review]

File: docs/developers/tutorials/crowdfunding-p1.md (hunk 2 above)
Comment (danielailie) [reply docs/developers/tutorials/crowdfunding-p1.md:75]: it was a problem with the tests from rust, I fixed it

**Pull request merged**